bundles/org.openhab.binding.ferroamp/src/main/java/org/openhab/binding/ferroamp/internal/EhubJsonElements.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ferroamp.internal;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link EhubJsonElements} is responsible for all parameters related to EHUB
+ *
+ * @author Örjan Backsell - Initial contribution
+ *
+ */
+
+@NonNullByDefault
+public class EhubJsonElements {
+    public String ehubJsonElements;
+
+    public EhubJsonElements(String ehubJsonElements) {
+        this.ehubJsonElements = ehubJsonElements;
+    }
+
+    public static List<String> getJsonElementsEhub() {
+        final List<String> jsonElementsEhub = new ArrayList<>();
+        jsonElementsEhub.add(new String("gridfreq"));
```

Comment:
Applies to all the cases where a string is added to a `List<String>`, so also in the `EsmJsonElements`, `EsoJsonElements` and `SsoJsonElements`

```suggestion
        jsonElementsEhub.add("gridfreq");
```

Reply:
Done described changes.
I thought I had to set type for every added element, (as I found in some example), but this is much better.